### ISSUE TITLE
Some modification for bVNC (option for UltraVNC connection)

### ISSUE THREAD

**Issue by Immick** [iiordanov/remote-desktop-clients#110]
Hello, Iordan Iordanov!

bVNC is the most powerful and comfortable Android client for VNC connection I've ever tried (believe me - I've tried a lot of such clients) - so I thank you for such wonderful application.

But there is one problem with using bVNC for me. I'm from Russia and it's really necessary to use Russain keyboard layout on server side. With BasicVNC connection type there is no problem with using Russian keyboard. But with UltraVNC connection type Russian keyboard is just ignoring (no input). On desktop PC UltraVNC client there is special option that solves problem with Russian (and I guess other) keyboards - this is "Japanese keyboard":

**[https://content.screencast.com/users/The_Immortal/folders/Snagit/media/78b81a12-8ab8-44f2-a18e-2134fd334286/09.02.2017-16.15.png](url)**

So when you enable this checkbox everything is good. But there is no such option on bVNC client.

And I beg you to add this option or find other way to solve this problem.

Thank you very much!

Regards,
Alex.

**Comment by iiordanov** [comment]
Indeed, there is a magical option in UltraVNC and the keymap is in keymapjap.h here:

http://cidis.ws/fecha_azul/UltraVNC/vncviewer/

I'll look into this magic when I get a chance. Thanks! :)
